ownCloud/Theming/ThemeCollection.swift

```diff
@@ -104,6 +105,11 @@ class ThemeCollection : NSObject {
 	@objc var darkBrandColors : ThemeColorCollection
 	@objc var lightBrandColors : ThemeColorCollection
 
+	// MARK: - Brand assets
+	@objc var brandedName : String? {
```

Comment:
`brandedName` does not seem to be used and I believe can therefore be removed. If `ownCloudSDK` was imported only for access to `OCAppIdentity` here, that `import ownCloudSDK` at the top of the file can be removed, too.

Reply:
Fixed